nsxt/resource_nsxt_policy_vni_pool.go

```diff
@@ -0,0 +1,164 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicyVniPool() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyVniPoolCreate,
+		Read:   resourceNsxtPolicyVniPoolRead,
+		Update: resourceNsxtPolicyVniPoolUpdate,
+		Delete: resourceNsxtPolicyVniPoolDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
Since multitenancy support, we added import by `path` to all resource for consistency (implemented via `nsxtPolicyPathResourceImporter`, it would still support import with ID).  I suggest we do the same here.

Reply:
Got it. Done.